com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -185,13 +180,34 @@ public class NetworkSceneManager : IDisposable
         /// When it is unset: After the scene has loaded, the PopulateScenePlacedObjects is called, and all NetworkObjects in the do
         /// not destroy temporary scene are moved into the active scene
         /// </summary>
-        internal static bool IsSpawnedObjectsPendingInDontDestroyOnLoad = false;
+        internal static bool IsSpawnedObjectsPendingInDontDestroyOnLoad;
```

Comment:
Even if default assignment is not needed, here, wouldn't having a default value help people understand the code and remove a potential confusion on the default value?

Reply:
The default value of a bool is false and it is non-standard to set the property of a bool to something it already defaults to.

Reply:
Yeah, I was the same way @jeffreyrainy but eventually I've gotten habituated to C#'s defaults